Core.ApplicationServices/UserService.cs

```diff
@@ -344,5 +371,26 @@ private static void Delete(User user)
             user.HasApiAccess = false;
             user.HasStakeHolderAccess = false;
         }
+
+        private Result<UserDeletionStrategy, OperationError> GetUserDeletionStrategy(int orgId, User user)
+        {
+            if (user.Deleted)
+                return new OperationError(OperationFailure.BadInput);
+            if (user.OrganizationRights.Any(x => x.OrganizationId == orgId) == false)
+                return new OperationError(OperationFailure.BadInput);
+
+            if (user.OrganizationRights.Any(x => x.OrganizationId != orgId))
+                return UserDeletionStrategy.Local;
+            if (!_organizationalUserContext.HasRoleInSameOrganizationAs(user))
+                return UserDeletionStrategy.Local;
+            if (_organizationalUserContext.HasRole(orgId, OrganizationRole.LocalAdmin))
+                return user.IsGlobalAdmin
+                    ? UserDeletionStrategy.Local
+                    : UserDeletionStrategy.Global;
+
+            return _organizationalUserContext.HasRole(orgId, OrganizationRole.GlobalAdmin)
+                ? UserDeletionStrategy.Global
+                : new OperationError(OperationFailure.Forbidden);
```

Comment:
now after removing the redindant code, if we get to this point then we know that:
- The current user is authorized to DELETE
- the "victim" is not already deleted
- The victim has a role in the org identified by org id
- The victim only has roles in the org identified by orgid

So here you will just `return Global`